.gitmodules

```diff
@@ -4,3 +4,6 @@
 [submodule "converter/COLLADA2GLTF/dependencies/OpenCOLLADA"]
 	path = converter/COLLADA2GLTF/dependencies/OpenCOLLADA
 	url = https://github.com/KhronosGroup/OpenCOLLADA.git
+[submodule "COLLADA2GLTF/dependencies/OpenCOLLADA"]
```

Comment:
Looks like the new location was added to `.gitmodules` without removing the old location?